### PR TITLE
feat: PowerShell support (frontend)

### PR DESCRIPTION
**Pull request by vaind**
This adds a new SDK to the frontend: PowerShell

I've implemented these changes by copying what is done for platform elixir.

I've tested the changes locally and everything seems to work fine AFAICT.

backend PR: https://github.com/getsentry/sentry/pull/67577

Resolves:
* https://github.com/getsentry/sentry-powershell/issues/43

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+const onboarding: OnboardingConfig = {
+  introduction: () =>
+    tct(
+      'Sentry for PowerShell module supports PowerShell 7.2+ on Windows, macOS, and Linux as well as Windows PowerShell 5.1+.',
+      {
+        strong: <strong />,
+        link: <ExternalLink href="https://docs.sentry.io/platforms/powershell/" />,
+      }
+    ),
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:71]:
```suggestion
    t(
      'Sentry for PowerShell module supports PowerShell 7.2+ on Windows, macOS, and Linux as well as Windows PowerShell 5.1+.'
    ),
```

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  install: params => [
+    {
+      type: StepType.INSTALL,
+      description: tct('Install the module:', {
+        strong: <strong />,
+      }),
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:77]:
```suggestion
      description: t('Install the module:'),
```

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+      ],
+    },
+  ],
+  configure: params => [
+    {
+      type: StepType.CONFIGURE,
+      description: tct('Initialize the SDK as early as possible.', {
+        sentrySdkCode: <code />,
+        programCode: <code />,
+      }),
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:99]:
```suggestion
      description: tct('Initialize the SDK as early as possible.'),
```

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        {
+          description: (
+            <List symbol="bullet">
+              <ListItem>
+                {tct('[link:Samples in the [code:powershell] SDK repository]', {
+                  link: (
+                    <ExternalLink href="https://github.com/getsentry/sentry-powershell/tree/main/samples" />
+                  ),
+                  code: <code />,
+                  strong: <strong />,
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:152]:
```suggestion
```

**Review by ArthurKnaus — COMMENTED** [review]
Nice work! 🙌 
We will need to merge the BE PR first. Otherwise creating a powershell project will fail.

**Comment by bruno-garcia** [comment]
the backend PR was merged so this is unblocked:
* https://github.com/getsentry/sentry/pull/67577

**Review by ArthurKnaus — APPROVED** [review]
👍

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  configure: params => [
+    {
+      type: StepType.CONFIGURE,
+      description: tct('Initialize the SDK as early as possible.'),
```
Comment (ArthurKnaus) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:90]: That is the typescript error, the tests run into:
```suggestion
      description: t('Initialize the SDK as early as possible.'),
```

File: static/app/gettingStartedDocs/powershell/powershell.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import {screen} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import docs from './powershell';
+
+describe('powershell onboarding docs', function () {
+  it('renders docs correctly', async function () {
+    renderWithOnboardingLayout(docs, {
+      releaseRegistry: {
+        'sentry.powershell': {
```
Comment (bruno-garcia) [inline static/app/gettingStartedDocs/powershell/powershell.spec.tsx:11]: see: https://github.com/getsentry/sentry-release-registry/pull/150

```suggestion
        'sentry.dotnet.powershell': {
```

File: static/app/gettingStartedDocs/powershell/powershell.tsx [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+      description: t('Install the module:'),
+      configurations: [
+        {
+          partialLoading: params.sourcePackageRegistries.isLoading,
+          code: [
+            {
+              language: 'powershell',
+              label: 'Install Module',
+              value: 'powershellget',
+              code: `Install-Module -Name Sentry -Repository PSGallery -RequiredVersion ${getPackageVersion(params, 'sentry.powershell', '1.0.0')} -Force`,
```
Comment (bruno-garcia) [inline static/app/gettingStartedDocs/powershell/powershell.tsx:80]:
```suggestion
              code: `Install-Module -Name Sentry -Repository PSGallery -RequiredVersion ${getPackageVersion(params, 'sentry.dotnet.powershell', '0.0.2')} -Force`,
```

**Comment by ArthurKnaus** [comment]
linting is still failing:
```
/home/runner/work/sentry/sentry/README.md
  4:7  error  Parsing error: JSX element 'img' has no corresponding closing tag
```

**Comment by vaind** [comment]
> linting is still failing:
> 
> ```
> /home/runner/work/sentry/sentry/README.md
>   4:7  error  Parsing error: JSX element 'img' has no corresponding closing tag
> ```

that has nothing to do with the changes in the PR, it's broken on the main branch. It's just that the check runs because this PR changes the file (on other lines)

**Comment by vaind** [comment]
Is there anything that needs to be done for this to get merged?

**Pull request merged**